examples/04_manipulating_images/plot_multiply_image.py

```diff
@@ -0,0 +1,21 @@
+"""
+Multiply image
+==============
+
+Here we multiply the voxel values of an image by -1.
+"""
+
+from nilearn import datasets, plotting, image
+
+data = datasets.fetch_atlas_aal()
```

Comment:
I think that it would speak more to the user if we used something else than the AAL atlas (why would we want to negate the atlas). How about a simple statistical map, as in
http://nilearn.github.io/auto_examples/01_plotting/plot_dim_plotting.html#sphx-glr-auto-examples-01-plotting-plot-dim-plotting-py